css/style.css

```diff
@@ -0,0 +1,74 @@
+body {
+  margin: 0;
+  background-color: #eee;
+}
+
+#header {
+  display: flex;
+}
+
+#nav-bar #header-img {
+  width: 500px;
+  margin-right: 200px;
+}
+
+#nav-bar {
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  background-color: burlywood;
+  width: 100%;
+}
+
+#nav-bar div {
+  display: inline-block;
+  margin: 20px;
+}
+
+#nav-bar .nav-link {
+  text-decoration: none;
+  color: black;
+}
+
+main {
+  margin-top: 10%;
+  text-align: center;
+}
+
+#submit {
+  padding: 10px;
+  margin: 10px;
+  background-color: #f1c410;
+  border: 0;
+}
+
+#allSections {
+  /* display: flex; */
+
+  /* justify-content: center; */
+
+  /* flex-direction: column; */
+
+  /* margin: 0 auto;
+   text-align: center; */
+
+  /* align-content:flex-end; */
+}
+
```

Comment:
I think you should not leave this commented code

Reply:
Try to explain "why" for each required change.